ansible_collections/arista/avd/roles/eos_snapshot/tasks/main.yml

```
@@ -39,7 +39,7 @@
 - name: save collected commands in text files
   template:
     src: device_text_command.j2
-    dest: "{{ snapshots_backup_dir }}/{{ inventory_hostname }}/{{ item.item }}.txt"
+    dest: "{{ snapshots_backup_dir }}/{{ inventory_hostname }}/{{ item.item | replace(\"/\", \"_\") }}.txt"
```

use single-quote for the inner quotes instead of escaping. This will make it more readable.
```suggestion
    dest: "{{ snapshots_backup_dir }}/{{ inventory_hostname }}/{{ item.item | replace('/', '_') }}.txt"
```

---

same for the markdown.yml

---

Thanks for the suggestion, fixed!